gr-utils/python/modtool/cli/add.py

```diff
@@ -101,15 +101,15 @@ def get_lang(self):
 
 def get_blockname(self):
     """ Get the blockname"""
-    if self.info['blockname'] is None:
+    if not self.info['blockname'] or self.info['blockname'].isspace():
         while not self.info['blockname'] or self.info['blockname'].isspace():
             self.info['blockname'] = cli_input("Enter name of block/code (without module name prefix): ")
     if not re.match('[a-zA-Z0-9_]+', self.info['blockname']):
```

Comment:
Strangely, I'm allowed to create a block with a space in the name (separate issue than this PR), but probably could be prevented here

Reply:
@mormj: Yes, that's right. What do you think would be better, replacing spaces with an underscore or disallowing the users to use a space in the `blockname`?

Reply:
Personally I think you could go either way - it's a very fringe case that someone would have to intentionally do with quotes.  Just disallowing is probably the safest.

Reply:
I think avoiding spaces is the way to go. I really don't see a real use-case where I would like to manage spaces, needless to say that they are handled different for different OS. I suggest just disallowing spaces